Makefile

```diff
@@ -31,10 +33,14 @@ build:
 	$(GO_BUILD_RECIPE)
 
 images/router/*/Dockerfile: images/router/base/Dockerfile
-	imagebuilder -t registry.svc.ci.openshift.org/openshift/origin-v4.0:`basename $(@D)`-router -f images/router/`basename $(@D)`/Dockerfile .
+	imagebuilder -t registry.ci.openshift.org/openshift/origin-v4.0:`basename $(@D)`-router -f images/router/`basename $(@D)`/Dockerfile .
```

Comment:
These Makefile changes were part of what I needed to get local router builds working, just fyi.

I didn't actually intend to push my changes to the Makefile, but since the existing Makefile references `registry.svc.ci.openshift.org` which has been retired for quite some time, I don't think it's being used in the official builds anyway. 

I'm happy to revert these changes and come back to fixing it later in another PR if that's preferable.

Reply:
> These Makefile changes were part of what I needed to get local router builds working, just fyi.

Then let's not commit them. I'd rather not mix make/build machinery as part of the fix if we don't have to.